### PR TITLE
Error handling in emoji connect flow

### PR DESCRIPTION
**Pull request by rzadp**

### CODE REVIEW

**Comment by rzadp** [comment]
## Before:

- `Cancel request` button not working

before-react-desktop
before-react-iphone
before-wallet-desktop
before-wallet-iphone

**Comment by rzadp** [comment]
## After:

- `Return` button working

after-react-desktop
after-react-iphone
after-jarvis-desktop
after-jarvis-iphone

**Comment by JustynaBroniszewska** [comment]
Tests are failing

File: universal-login-react/src/ui/ConnectionFlow/ConnectWithEmoji.tsx [hunk 1]
```diff
@@ -36,22 +36,31 @@ export const ConnectWithEmoji = ({name, sdk, onCancel, onConnect, walletService,
...
-          </div>
+          {!error && <TitledContent title='Confirmation'>
+            <p className="connect-emoji-text">Check the notification of another device controlling this account and type the emojis in this order.</p>
+            {!connectValues && <Spinner className="spinner-center" />}
+            {connectValues && <div className="universal-login-emojis">
+              <EmojiPanel className="jarvis-styles" code={connectValues!.securityCode} />
+            </div>}
+            <button onClick={onCancelClick} className="connect-emoji-btn">Cancel Request</button>
+          </TitledContent>}
+          {error && <TitledContent title='Something went wrong.'>
```
Comment (JustynaBroniszewska) [inline universal-login-react/src/ui/ConnectionFlow/ConnectWithEmoji.tsx]: I would propose to extract more generic for example ErrorMessageScreen that we can use everywhere

Comment (JustynaBroniszewska) [reply universal-login-react/src/ui/ConnectionFlow/ConnectWithEmoji.tsx]: Then we don't need to extract TitledContent

**Comment by rzadp** [comment]
WIP - working on separating default styling now

File: universal-login-react/src/ui/commons/ErrorMessage.tsx [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+interface ErrorMessageProps {
+  title?: string;
+  message?: string;
+  children?: ReactNode;
+  className?: string;
+}
+
+export const ErrorMessage = ({title = 'Something went wrong', message = 'Please try again later.', children, className}: ErrorMessageProps) => (
+  <div className={getStyleForTopLevelComponent(className)}>
```
Comment (alexcherman) [inline universal-login-react/src/ui/commons/ErrorMessage.tsx]: Wrap it in ``<div className="universal-login-error-message">...</div>``

File: universal-login-react/src/ui/styles/errorMessage.sass [hunk 3]
```diff
@@ -0,0 +1,15 @@
+.error-message-title
```
Comment (alexcherman) [inline universal-login-react/src/ui/styles/errorMessage.sass]: Add .universal-login-error-message on top

File: universal-login-react/src/ui/styles/errorMessageDefault.sass [hunk 4]
```diff
@@ -0,0 +1,7 @@
+.universal-login-default
```
Comment (alexcherman) [inline universal-login-react/src/ui/styles/errorMessageDefault.sass]: Add .universal-login-error-message on top

File: universal-login-wallet/src/ui/styles/layout/_error-message.sass [hunk 5]
```diff
@@ -0,0 +1,17 @@
+.error-message-title
```
Comment (alexcherman) [inline universal-login-wallet/src/ui/styles/layout/_error-message.sass]: Add .jarvis-styles on top

File: universal-login-react/src/ui/commons/ErrorMessage.tsx (hunk 2 above)
Comment (rzadp) [reply universal-login-react/src/ui/commons/ErrorMessage.tsx]: Done.

File: universal-login-react/src/ui/styles/errorMessage.sass (hunk 3 above)
Comment (rzadp) [reply universal-login-react/src/ui/styles/errorMessage.sass]: Done.

File: universal-login-react/src/ui/styles/errorMessageDefault.sass (hunk 4 above)
Comment (rzadp) [reply universal-login-react/src/ui/styles/errorMessageDefault.sass]: Done.

File: universal-login-wallet/src/ui/styles/layout/_error-message.sass (hunk 5 above)
Comment (rzadp) [reply universal-login-wallet/src/ui/styles/layout/_error-message.sass]: Done.

**Pull request merged**